index.js

```diff
@@ -12,8 +14,10 @@ const obj = () => Object.create(null);
 delete require.cache[__filename];
 const parentDir = path.dirname((module.parent && module.parent.filename) || '.');
 
-class Conf {
+class Conf extends EventEmitter {
```

Comment:
There's no point in extending EventEmitter if we're not going to use its methods.

Reply:
I had it emitting a `change` event when the store was updated (L118), but that's really only about it.
Otherwise we could potentially store all watch functions in a local array and call them from there.

Reply:
We could just simply hook into `config.set` and then call the `onDidChange` callbacks from there?

Reply:
Just init a new EventEmitter instance in the Conf init constructor and use it internally as a property.

Reply:
👍 I'll do that.